Tests/SonarScanner.MSBuild.Tasks.IntegrationTest/TargetsTests/RazorTargetTests.cs

```diff
@@ -40,7 +40,7 @@ public class RazorTargetTests
         public TestContext TestContext { get; set; }
 
         [TestMethod]
-        public void Razor_CheckErrorLogProperties()
+        public void Razor_WhenNoSonarErrorLog_NoPropertiesAreSet()
```

Comment:
WDYT about adding an assert that SonarTemporaryProjectSpecificOutDir was set in this case too?